packages/react/src/VideoPlayer/VideoPlayer.module.css

```
@@ -48,20 +47,21 @@
   z-index: 1;
 }
 
-.VideoPlayer__playButtonOverlay {
+.VideoPlayer__playButtonOverlay.VideoPlayer__playButtonOverlay--transparent {
+  background: transparent;
+}
+
+.VideoPlayer__playButtonOverlay:focus-visible {
+  outline: var(--brand-borderWidth-thicker) solid var(--brand-color-focus);
+  outline-offset: 2px;
```

We have a 2px base token now 😌 

```suggestion
  outline-offset: var(--base-size-2);
```